test/EVMHost.cpp

```diff
@@ -35,17 +35,17 @@ using namespace dev;
 using namespace dev::test;
 
 
-evmc::vm* EVMHost::getVM(string const& _path)
+evmc::VM* EVMHost::getVM(string const& _path)
 {
-	static unique_ptr<evmc::vm> theVM;
+	static unique_ptr<evmc::VM> theVM;
 	if (!theVM && !_path.empty())
 	{
 		evmc_loader_error_code errorCode = {};
-		evmc_instance* vm = evmc_load_and_configure(_path.c_str(), &errorCode);
+		evmc_vm* vm = evmc_load_and_configure(_path.c_str(), &errorCode);
```

Comment:
```suggestion
		theVM = evmc::VM{evmc_load_and_configure(_path.c_str(), &errorCode)};
```
where `theVM` is of `evmc::VM` type.